### PR TITLE
Added Ent chopping animation to AnimationID list.

### LINKED ISSUE #12898: Ent idle notifier
**Describe the bug**
The idle notifier doesn't notify when done chopping an ent trunk

**To Reproduce**
Kill an ent and chop it's trunk while the idle notifier plugin is on

**Expected behavior**
Notify when done chopping

**Environment (please complete the following information):**
 - OS: Ubuntu (pretty sure it happens on Windows as well)
 - RuneLite version: 1.6.33
 - Launcher version: 2.1.3

I also noticed that the loot tracker plugin doesn't work for ent trunks, is this intended?

### PR DESCRIPTION
**Pull request by mcdogerts**
Also added case for Ent trunk animation in idle notifier plugin. Verified idle notifier working when idle after chopping ent trunk.

Closes #12898

### CODE REVIEW

**Comment by makingcomment** [comment]
The idle notifier additions seem to be missing the crystal/trailblazer axes. Is that intentional?

**Comment by mcdogerts** [comment]
I couldn't verify that they worked properly, so I did not include them.

**Comment by Nightfirecat** [comment]
This needs rebase. Also, the standard trailblazer animation IDs are present in #13019, so the crystal/trailblazer/infernal trailblazer IDs should be fairly easy to find by using `::anim` via the dev tools, even if you don't own the tools themselves.

File: runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/Axe.java [hunk 1]
```diff
@@ -85,15 +80,18 @@
...
+			}
 		}
 
 		AXE_ANIM_IDS = builder.build();
 	}
 
 	boolean matchesChoppingAnimation(final Player player)
 	{
-		return player != null && animId == player.getAnimation();
+		return player != null && findAxeByAnimId(player.getAnimation()) == this;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/Axe.java:94]:
```suggestion
		return player != null && this == findAxeByAnimId(player.getAnimation());
```

**Comment by Batsju** [comment]
> Also added case for Ent trunk animation in idle notifier plugin. Verified idle notifier working when idle after chopping ent trunk.
> 
> Closes #12898

How can i add this code to my RuneLite app? would love to have this notifier.

**Comment by Nightfirecat** [comment]
Closing as this PR is stale.

**Pull request closed without merging**